spec/views/organisations/form.html.erb_spec.rb

```diff
@@ -37,19 +37,20 @@
   end
 
   it 'should have categories in scroll box ordered by type and name' do
-    expect(rendered).to have_xpath("//div/strong/em[text()='What you do']/../../following-sibling::div[1]/label[text()='alligator']")
-    expect(rendered).to have_xpath("//div/strong/em[text()='What you do']/../../following-sibling::div[2]/label[text()='capybara']")
-    expect(rendered).to have_xpath("//div/strong/em[text()='Who you help']/../../following-sibling::div[1]/label[text()='crocodile']")
-    expect(rendered).to have_xpath("//div/strong/em[text()='Who you help']/../../following-sibling::div[2]/label[text()='guinea pig']")
-    expect(rendered).to have_xpath("//div/strong/em[text()='How you help']/../../following-sibling::div[1]/label[text()='iguana']")
-    expect(rendered).to have_xpath("//div/strong/em[text()='How you help']/../../following-sibling::div[2]/label[text()='rabbit']")
+    expect(rendered).to have_xpath("//h5[contains(., 'What you do')]/following-sibling::div[1]/label[text()[contains(.,'alligator')]]")
```

Comment:
hehehe, I have managed to do it without adding a class to h5. Let me know please if I shouldn't do it like that.
Also I was wondering whether we could change the it statement to remove in scroll box, as it's not in a scroll box anymore.